templates/manila/config/logging.conf

```diff
@@ -0,0 +1,34 @@
+[loggers]
```

Comment:
Imported by glance-operator: @gouthampacha feel free to propose any improvement on this file too!

Reply:
This looks good for now; we'll follow up by lnking debug logging to the log level config on line 25, alongside setting appropriate [library log settings](https://docs.openstack.org/oslo.log/latest/configuration/#DEFAULT.default_log_levels)